engine/src/main/scala/cromwell/engine/workflow/lifecycle/InitializeWorkflowBackendsActor.scala

```diff
@@ -0,0 +1,114 @@
+package cromwell.engine.workflow.lifecycle
+
+import akka.actor.{ActorRef, LoggingFSM, Props}
+import cromwell.backend.BackendWorkflowInitializationActor._
+import cromwell.engine.workflow.lifecycle.InitializeWorkflowBackendsActor._
+import wdl4s.Call
+
+object InitializeWorkflowBackendsActor {
```

Comment:
Okay this was confusing to me... we have a `InitializeWorkflowBackendsActor` and a `BackendWorkflowInitializationActor`... same 4 words... I'd certainly not be opposed to some better terminology